build/SignConfig.xml

```diff
@@ -2,13 +2,13 @@
 <SignConfigXML>
   <job platform="" configuration="" dest="__INPATHROOT__" jobname="dep.controls" approvers="">
     <!-- Overwrite the input files to simplify downstream things like nuget and framework package generation. -->
-    <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
-    <file src="__INPATHROOT__\Release\x64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\x64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\x64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\x64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
-    <file src="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
-    <file src="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
+    <file src="__INPATHROOT__\runtimes\win10-x86\native\Microsoft.ApplicationModel.Resources.dll" signType="WindowsRT" dest="__INPATHROOT__\runtimes\win10-x86\native\Microsoft.ApplicationModel.Resources.dll" />
```

Comment:
><file [](start = 4, length = 5)

compare with Singconfig.xml for MRT, it misses a few files.

Reply:
right, looks like this needs mrm.dll at minimum.  Looks good overall, though, thanks.

Reply:
We don't need to sign the lib or pdb files correct?

Reply:
yes, that is correct, just dll and winmd.

Reply:
@huichen123, is mrm.dll supposed to be in this list?

Reply:
yes, all dlls need signed. still missing winmd and Microsoft.ApplicationModel.Resources.Projection.dll.

Reply:
It should have all the files now, the winmd and both copies of the projection dlls.